src/web/core/src/test/java/org/geoserver/web/data/layer/LayerPageTest.java

```diff
@@ -82,7 +82,8 @@ public void testBasicActions() {
         assertEquals("cite", workspaces.get(1));
 
         // select second layer
-        String checkBoxPath = "table:listContainer:items:6:selectItemContainer:selectItem";
+        // a better way to get "table:listContainer:items:6:selectItemContainer:selectItem";
+        String checkBoxPath = getNthComponentPath(table, CheckBox.class, 1);
         CheckBox selector = (CheckBox) tester.getComponentFromLastRenderedPage(checkBoxPath);
         // dirty trick, how to set a form component value without a form
         tester.getRequest().setParameter(selector.getInputName(), "true");
```

Comment:
Can anyone why this would start to fail with a wicket upgrade - it makes no difference which version is generating the page to be tested (I think) so I suspect it is a "dirty hack" that has stopped working.

Reply:
Hum... duh not sure, but think the paths that wicket is generating has changed. Maybe there is more or less intermediate containers in the mix? Not sure.

Reply:
copied a fix from the WPS tests that seems to work